CMakeLists.txt

```diff
@@ -681,11 +664,10 @@ endif ()
 #-----------------------------------------------------------------------------
 # Add some definitions for Debug Builds
 #-----------------------------------------------------------------------------
-option (HDF5_ENABLE_TRACE "Enable API tracing capability" OFF)
-mark_as_advanced (HDF5_ENABLE_TRACE)
 if (${HDF_CFG_NAME} MATCHES "Debug" OR ${HDF_CFG_NAME} MATCHES "Developer")
   # Enable instrumenting of the library's internal operations
-  cmake_dependent_option (HDF5_ENABLE_INSTRUMENT "Instrument The library" OFF HDF5_ENABLE_TRACE OFF)
+  option (HDF5_ENABLE_INSTRUMENT "Instrument The library" OFF)
+  mark_as_advanced (HDF5_ENABLE_INSTRUMENT)
 
```

Comment:
Is there a reason for this (and some others) to be added here instead of the new file?